app/services/support_interface/equality_and_diversity_export.rb

```diff
@@ -10,12 +10,12 @@ def data_for_export
           'Ethnic group' => application_form.equality_and_diversity['ethnic_group'],
           'Ethnic background' => application_form.equality_and_diversity['ethnic_background'],
           'Application status' => I18n.t!("candidate_flow_application_states.#{ProcessState.new(application_form).state}.name"),
-          'First rejection reason' => rejected_application_choices[0]&.rejection_reason,
-          'Second rejection reason' => rejected_application_choices[1]&.rejection_reason,
-          'Third rejection reason' => rejected_application_choices[2]&.rejection_reason,
-          'First structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[0]&.structured_rejection_reasons),
-          'Second structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[1]&.structured_rejection_reasons),
-          'Third structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[2]&.structured_rejection_reasons),
+          'First application unstructured rejection reason' => rejected_application_choices[0]&.rejection_reason,
+          'Second application unstructured rejection reason' => rejected_application_choices[1]&.rejection_reason,
+          'Third application unstructured rejection reason' => rejected_application_choices[2]&.rejection_reason,
+          'First application structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[0]&.structured_rejection_reasons),
+          'Second application structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[1]&.structured_rejection_reasons),
+          'Third application structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[2]&.structured_rejection_reasons),
```

Comment:
Hey man, looking good! sorry to nit pick but i think we should make it clear that each column is an 'application choice' as opposed to an application as each row of the table currently represents one application. to my mind it would look something like:
```suggestion
          'Application choice 1 unstructured rejection reason' => rejected_application_choices[0]&.rejection_reason,
          'Application choice 2 unstructured rejection reason' => rejected_application_choices[1]&.rejection_reason,
          'Application choice 3 unstructured rejection reason' => rejected_application_choices[2]&.rejection_reason,
          'Application choice 1 structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[0]&.structured_rejection_reasons),
          'Application choice 2 structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[1]&.structured_rejection_reasons),
          'Application choice 3 structured rejection reasons' => format_structured_rejection_reasons(rejected_application_choices[2]&.structured_rejection_reasons),
```
What do you think?